src/webui/www/private/views/preferences.html

```diff
@@ -1984,6 +2008,8 @@
             window.parent.qBittorrent.Cache.preferences.init({
                 onSuccess: (pref) => {
                     // Behavior tab
+                    $("dblclick_download_select").value = LocalPreferences.get("dblclick_download", "1");
```

Comment:
Actually I just read

> The MooTools library has seen little maintenance and therefore its usage is discouraged.

Should I replace `$` with native JS methods? Or better to keep it for consistency with the surrounding code?

Reply:
I think you can keep `$` for now. I haven't decided to replace `$` globally.